server/konnectors/freemobile.coffee

```diff
@@ -0,0 +1,348 @@
+americano = require 'americano-cozy'
+requestJson = require 'request-json'
+
+moment = require 'moment'
+cheerio = require 'cheerio'
+fs = require 'fs'
+async = require 'async'
+pngjs = require 'pngjs-image'
+
+
+
+File = require '../models/file'
+fetcher = require '../lib/fetcher'
+filterExisting = require '../lib/filter_existing'
+saveDataAndFile = require '../lib/save_data_and_file'
+linkBankOperation = require '../lib/link_bank_operation'
+
+localization = require '../lib/localization_manager'
+
+log = require('printit')
+    prefix: "Free Mobile"
+    date: true
+#Useragent is required
+request = require 'request'
+request = request.defaults
+    headers:
+        "User-Agent": "Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:37.0) Gecko/20100101 Firefox/37.0"
+
+# Models
+
+PhoneBill = americano.getModel 'PhoneBill',
+    date: Date
+    vendor: String
+    amount: Number
+    fileId: String
+
+PhoneBill.all = (callback) ->
+    PhoneBill.request 'byDate', callback
+
+# Konnector
+
+module.exports =
+
+    name: "Free Mobile"
+    slug: "freemobile"
+    description: 'konnector description free mobile'
+    vendorLink: "https://mobile.free.fr/"
+
+    fields:
+        login: "text"
+        password: "password"
+        folderPath: "folder"
+    models:
+        phonebill: PhoneBill
+
+    # Define model requests.
+    init: (callback) ->
+        map = (doc) -> emit doc.date, doc
+        PhoneBill.defineRequest 'byDate', map, (err) ->
+            callback err
+
+    fetch: (requiredFields, callback) ->
+
+        log.info "Import started"
+
+        fetcher.new()
```

Comment:
It's great that you used the Fetcher. Thank you for taking the time to understand how it works. That's hard without documentation.